assets/js/modules/analytics-4/components/dashboard/ActivationBanner/ReminderBannerNoAccess.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * ReminderBannerNoAccess component.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import BannerNotification from '../../../../../components/notifications/BannerNotification';
+import { CORE_MODULES } from '../../../../../googlesitekit/modules/datastore/constants';
+
+const { useSelect } = Data;
+
+export default function ReminderBannerNoAccess( {
+	title,
+	description,
+	descriptionIcon,
+	dismissExpires,
+	onDismiss,
+} ) {
+	const module = useSelect( ( select ) =>
+		select( CORE_MODULES ).getModule( 'analytics' )
+	);
+
+	const formattedOwnerName = module?.owner?.login
+		? `<strong>${ module.owner.login }</strong>`
+		: __( 'Another admin', 'google-site-kit' );
+
+	description = createInterpolateElement(
+		description +
+			'. <br />' +
+			sprintf(
+				/* translators: 1: module owner's name, 2: module name */
+				__(
+					'%1$s configured Analytics and you don’t have access to this Analytics property. Contact them to share access or change the Analytics property.',
+					'google-site-kit'
+				),
+				formattedOwnerName
+			),
+		{
+			strong: <strong />,
+			br: <br />,
+		}
```

Comment:
I'm confused by this bit—why isn't it also translated? We shouldn't be using non-localisable strings as part of user-facing text like this, because if the formatting/punctuation should be localised we can't do it here.

Seems like this is happening because the description won't (or might not?) have a full-stop, but this seems a hacky/fragile workaround. It'd be better to remove this, only should a line break, and integrate the `description` into the `sprintf()`'d string:

```suggestion
			sprintf(
				/* translators: 1. Original reminder description text, 2: module owner's name */
				__(
					'%1$s<br />%2$s configured Analytics and you don’t have access to this Analytics property. Contact them to share access or change the Analytics property.',
					'google-site-kit'
				),
				description,
				formattedOwnerName
			),
		{
			strong: <strong />,
			br: <br />,
		}
```

(Also, the `translators` description of this string is off, there was only one placeholder. I've removed the module name in the suggestion here.)